source/common/filter/config_discovery_impl.h

```diff
@@ -65,21 +126,11 @@ class DynamicFilterConfigProviderImpl : public DynamicFilterConfigProvider {
     absl::optional<Envoy::Http::FilterFactoryCb> config_{};
   };
 
-  FilterConfigSubscriptionSharedPtr subscription_;
-  const absl::flat_hash_set<std::string> require_type_urls_;
   // Currently applied configuration to ensure that the main thread deletes the last reference to
   // it.
-  absl::optional<Envoy::Http::FilterFactoryCb> current_config_{absl::nullopt};
+  absl::optional<Envoy::Http::FilterFactoryCb> current_config_;
```

Comment:
nit: I'd probably keep it as it is to indicate `current_config_` isn't going to be inited in ctor.